presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -278,7 +278,8 @@ public static QueryStats create(
             DataSize peakTotalMemoryReservation,
             DataSize peakTaskUserMemory,
             DataSize peakTaskTotalMemory,
-            DataSize peakNodeTotalMemory)
+            DataSize peakNodeTotalMemory,
+            Optional<RuntimeStats> runtimeStats)
```

Comment:
Could this Optional be removed? The runtimeStats is always present in the Session, right?